LICENSE-MIT

```diff
@@ -0,0 +1,23 @@
+Permission is hereby granted, free of charge, to any
+person obtaining a copy of this software and associated
+documentation files (the "Software"), to deal in the
+Software without restriction, including without
+limitation the rights to use, copy, modify, merge,
+publish, distribute, sublicense, and/or sell copies of
+the Software, and to permit persons to whom the Software
+is furnished to do so, subject to the following
+conditions:
+
+The above copyright notice and this permission notice
```

Comment:
There needs to be an above copyright notice for this to make sense.

(This is a difference between the Apache and MIT licenses. For Apache, the copyright line should go [in a separate NOTICE file](https://www.apache.org/licenses/LICENSE-2.0#apply); for MIT, it should go directly above the license text.)